docs/commands.md

```diff
@@ -822,6 +822,20 @@ GRAPH.QUERY DEMO_GRAPH
 "MATCH (:Employer {name: 'Dunder Mifflin'})-[:EMPLOYS]->(p:Person) RETURN p"
 ```
 
+RedisGraph will use one index per scan operation, always trying to select the best index by checking the type of comparisons being made. In the example below the Name index will be used because the `=` comparison is likely to produce far fewer results than the `>` comparison.
+
+```sh
+GRAPH.QUERY DEMO_GRAPH
+"MATCH (p:Person) WHERE p.name = "Steven Tucker" AND p.age > 30 RETURN p"
```

Comment:
change to single quote string
`"MATCH (p:Person) WHERE p.name = 'Steven Tucker' AND p.age > 30 RETURN p"`

also the statement isn't true, if both `name` and `age` attribute of `Person` are indexed, then this query will use the index for both:

```
GRAPH.QUERY g "create index on :Person(age)"
GRAPH.QUERY g "create index on :Person(name)"
127.0.0.1:6379> GRAPH.EXPLAIN g "MATCH (p:Person) WHERE p.name = 'Steven Tucker' AND p.age > 30 RETURN p"
1) "Results"
2) "    Project"
3) "        Index Scan | (p:Person)"
127.0.0.1:6379>
```

Reply:
Oh, this was my mistake. Sorry, @elena-kolevska!

Reply:
No problem at all @jeffreylovitz  :)
The part about multiple indexes used to be documented (still online for version 2.2) but has since been removed, so I think I will just close this PR, unless you think we should add it back in.